python/pyspark/sql/readwriter.py

```diff
@@ -33,10 +33,12 @@
 
 def to_str(value):
     """
-    A wrapper over str(), but convert bool values to lower case string
+    A wrapper over str(), but convert bool values to lower case string, and keep None
```

Comment:
hm i think you need to expand a little bit more and explain what "keep None" means.